docs/operate/reference/components/sensor/lorawan.md

```diff
@@ -0,0 +1,519 @@
+| `fport` | string | Optional | 8-bit hexadecimal **frame port** used to send downlinks to the device. Found in the device datasheet. <br> On supported Milesight models, defaults to `"55"` (equivalent to `0x55`). <br> On supported Dragino models, defaults to `"01"` (equivalent to `0x01`). |
+
+{{% alert title="Info" color="info" %}}
+
+Device-specific models, including those for Milesight and Dragino sensors, provide default values for `app_key`, `network_s_key`, `app_s_key`, `fport`, `decoder_path`, and `uplink_interval_mins`.
+
+{{% /alert %}}
+
+### Activation protocols
+
+LoRaWAN networks can use any of the following protocols for communication:
+
+<!-- prettier-ignore -->
+| Name | Value | Key | Session Key | Configuration |
+| ---- | ----- | --- | ----------- | ------------- |
+| Over-The-Air Activation | `OTAA` | Dynamic, generated at join time | Automatically rotated | <ul><li> `dev_eui` </li><li> `app_eui` </li><li> `app_key` </li></ul> |
+| Activation By Personalization | `ABP` | Static | Static unless manually rotated | <ul><li> `dev_addr` </li><li> `network_s_key` </li><li> `app_s_key` </li></ul> |
+
+To specify an activation protocol for your network, use the `join_type` field.
+Many nodes use the OTAA protocol by default.
+
+### Decoder script
+
+When a LoRaWAN device transmits data, it sends compressed binary payloads to minimize power consumption and airtime.
+Decoder scripts convert this binary data into structured, human-readable formats like JSON.
+
+Each manufacturer uses different encoding schemes depending on the data transmitted by a device.
+For example:
+
+- A temperature sensor might encode 23.5°C as a hexadecimal value like `0x00EB`
+- A GPS tracker might pack latitude and longitude into 8 bytes.
+
+Without a decoder script, your application receives meaningless byte sequences like `0x00EB1337` instead of useful data like `{"temperature": 23.5, "humidity": 42.0 }`.
+
+Device manufacturers typically provide scripts for each device. For examples, see the following GitHub repositories:
+
+- [Milesight](https://github.com/Milesight-IoT/SensorDecoders)
+- [Dragino](https://github.com/dragino/dragino-end-node-decoder)
+
+### Frame port
+
+Frame port (`fport`) is an 8-bit field in the LoRaWAN MAC payload structure that defines the data type contained within the frame payload.
+Frame ports identify the kind of data passed in an uplink or downlink message so it can be routed to the correct handler, which helps keep LoRaWAN traffic fast, simple, and efficient.
+
+Use the `fport` field of a node configuration to specify the frame port value that the node expects to use for downlink communication.
+When you specify an `fport` value for a node, gateways use that value as the frame port for all downlink communication with the node.
+
+## Control nodes directly
```

Comment:
Naively, I find myself wondering "wait, did I just miss something about controlling them indirectly?" I also find myself wondering "Are there do commands to control the gateway?" Maybe add a sentence transitioning into this section?

Otherwise this seems like a very nice user-friendly section :D

Reply:
I don't think there are any gateway docommands, but I will double check that!

Reply:
If not, [opt] add something to this transition that says so, along with the direct vs indirect bit